plugins/node/opentelemetry-instrumentation-cassandra/test/cassandra-driver.test.ts

```diff
@@ -64,6 +67,13 @@ function assertSpan(
     attributes[SemanticAttributes.DB_STATEMENT] = query;
   }
 
+  if (customAttributes) {
+    customAttributes.forEach(customAttribute => {
```

Comment:
Instead of the `if` you can write `customAttributes?.forEach` which is a bit cleaner.